### PR TITLE
did a quick cleanup on indentations and formatting

### PR DESCRIPTION
**Pull request by LancePants97**
## PR Checklist
- [x] Added Reviewer
- [ ] Followed TDD, And Test are Passing
- [ ] Linked Issues for Project Board

### CODE REVIEW

File: app/views/dashboard/index.html.erb [hunk 1]
```diff
@@ -1,64 +1,64 @@
...
-            </li>
-            <li class="nav-item" role="presentation" id="item-tab" data-bs-toggle="tab" data-bs-target="#item" role="tab" aria-controls="item" aria-selected="true">
-                <a class="nav-link" href="<%=merchant_items_path(@merchant)%>">My Items</a>
-            </li>
-            <li class="nav-item" role="presentation" id="invoice-tab" data-bs-toggle="tab" data-bs-target="#invoice" role="tab" aria-controls="invoice" aria-selected="true">
-                <a class="nav-link" href="<%= merchant_invoices_path(@merchant) %>">My Invoices</a>
-            </li>
-        </ul>
-    </div>
+  <div class="row" id="page_info">
```
Comment (JCL461437) [inline app/views/dashboard/index.html.erb:2]: Looks great!

File: spec/features/dashboard/index_spec.rb [hunk 2]
```diff
@@ -1,47 +1,42 @@
...
-            within("#items_to_ship") do
-                expect(page).to have_content("Items Ready To Ship")
-                expect(page).to have_content("Item Name")
-                expect(page).to have_content("Invoice Number")
-                expect(page).to have_content("Date")
-                # add data to check in table
-                
-            end
+    end
+    scenario "I see a div for items ready to ship" do
```
Comment (JCL461437) [inline spec/features/dashboard/index_spec.rb:31]: Wow you were very thorough with this. Great work!

**Review by JCL461437 — APPROVED** [review]
Great work Lance. That must have taken some time

**Review by austincarrjones — APPROVED** [review]
nice, thanks for adjusting

**Pull request closed without merging**